app/controllers/ConfigurationController.scala

```diff
@@ -1,27 +1,25 @@
 package controllers
 
+import com.scalableminds.util.tools.Fox
 import javax.inject.Inject
-
 import models.binary.{DataSet, DataSetDAO, DataSetSQLDAO}
 import models.configuration.{DataSetConfiguration, UserConfiguration}
-import models.user.UserService
-import oxalis.security.WebknossosSilhouette.{SecuredAction, SecuredRequest, UserAwareAction, UserAwareRequest}
+import models.team.OrganizationSQLDAO
+import models.user.{UserDataSetConfigurationSQLDAO, UserService}
+import oxalis.security.WebknossosSilhouette.{SecuredAction, UserAwareAction}
 import play.api.i18n.{Messages, MessagesApi}
 import play.api.libs.concurrent.Execution.Implicits._
-import play.api.libs.json.JsObject
+import play.api.libs.json.{JsObject, JsValue}
 import play.api.libs.json.Json._
-import play.api.mvc.Action
-import play.libs.Json
 
-/**
- * Controller that handles the CRUD api for configurations (mostly settings for the tracing view)
- */
+
 class ConfigurationController @Inject()(val messagesApi: MessagesApi) extends Controller {
 
   def read = UserAwareAction.async { implicit request =>
     request.identity.toFox.flatMap { user =>
-      UserService.findOneById(user.id, useCache = false)
-      .map(_.userConfiguration.configurationOrDefaults)
+      for {
```

Comment:
just out of curiosity: why did you switch here to the `for` syntax?

Reply:
userConfigurationStructured now returns a Fox instead of the content directly (this is a common change in this PR). I tend to get confused with nested `flatMaps`, which would have now been necessary here.

Reply:
got it, thanks :+1: